schemas/2014-08-01/Microsoft.Scheduler.json

```diff
@@ -178,38 +243,58 @@
       "type": "object",
       "properties": {
         "uri": {
-          "type": "string",
-          "maxLength": 2048,
-          "description": "Microsoft.Scheduler/jobCollections/jobs: URI of the endpoint to call"
-        },
-        "method": {
           "anyOf": [
```

Comment:
My thought was that if the expression ended up being longer than 2048 characters, then they would get schema errors. It's unlikely, but I decided it was possible.
If you feel the odds are too low to justify this, then I'll remove the expression clause of this "anyOf".